### ISSUE TITLE
versions manifest file merge conflict when using separate pull requests

### ISSUE THREAD

**Issue by tpolekhin** [google-github-actions/release-please-action#526]
### TL;DR

When using `"separate-pull-requests": true` with multi-package release-please config you can have 2 chore PRs opened at the same time as a result of 2 different changes in different packages, but they both update `.release-please-manifest.yaml` file.
When you merge one of the PRs to trigger a version release other PR is left in the merge conflict state

### Expected behavior

You should be able to have multiple chore PRs opened at the same time in a multi-package project and be able to release them independently without merge conflicts

### Observed behavior

When you merge one of the PRs to trigger a version release other PR is left in the merge conflict state

### Action YAML

```yaml
name: Release Please

on:
  push:
    branches:
      - main

jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          command: manifest
```

### Log output

```text
https://github.com/tpolekhin/release-please-test/runs/7360900306
```

### Additional information

link the conflict PR: https://github.com/tpolekhin/release-please-test/pull/11

release-please-config.json
```json
{
  "release-type": "terraform-module",
  "separate-pull-requests": true,
  "packages": {
    "modules/eks": {
      "package-name": "eks"
    },
    "modules/vpc": {
      "package-name": "vpc"
    }
  }
}
```
.release-please-manifest.json
```
{
  "modules/eks": "0.2.0",
  "modules/vpc": "0.1.2"
}
```

**Comment by chingor13** [comment]
Needs the upstream feature request

**Comment by StefanBrand** [comment]
Also related, with workaround of prettifying manifest: https://github.com/googleapis/release-please/issues/1870

**Comment by shinebayar-g** [comment]
Got a workaround working without needing to insert filler lines. See https://github.com/googleapis/release-please/issues/1870#issuecomment-1748390833
